### DISCUSSION TITLE
DNGlab on MacOS?

### DISCUSSION THREAD

**Discussion by devonstanczyk** [dnglab/dnglab#419] [General]
Is DNGlab possible to run on MacOS? If so, whats the installation and running instructions?

**Comment by cytrinox** [comment]
You need to build it from source, for example using https://rustup.rs/
The CI/CD pipeline has no build target for MacOS yet. I don't have any experience with Apple/Mac. PRs welcome :smile:

**Reply by devonstanczyk** [reply]
I will be testing! Thank you! - After installing Rustup, what command should I use to target the latest merges?

**Comment by devonstanczyk** [comment]
Tested on MacOS. Working well if you point to the binary 👍
